db/database_test.go

```diff
@@ -388,7 +388,10 @@ func TestGetRemovedAsUser(t *testing.T) {
 	// Manually remove the temporary backup doc from the bucket
 	// Manually flush the rev cache
 	// After expiry from the rev cache and removal of doc backup, try again
-	db.DatabaseContext.revisionCache = NewShardedLRURevisionCache(KDefaultRevisionCacheCapacity, db.DatabaseContext.revCacheLoader, db.DatabaseContext.DbStats.StatsCache())
+	revCacheOptions := RevisionCacheOptions{
+		Size: KDefaultRevisionCacheCapacity,
+	}
+	db.DatabaseContext.revisionCache = NewShardedLRURevisionCache(&revCacheOptions, db.DatabaseContext.revCacheLoader, db.DatabaseContext.DbStats.StatsCache())
```

Comment:
For these tests I guess we can pass nil in, as `NewShardedLRURevisionCache` will use the default constants if unset.

Reply:
Very true.